packages/runtime/test/src/interpreter/inner-transaction.ts

```diff
@@ -569,9 +561,12 @@ describe("TEALv5: Inner Transactions", function () {
     // TODO: check if this should indeed fail (if receiver account.balance < minBalance)
     it(`should pass: after increasing app account's balance`, function () {
       const acc = interpreter.runtime.ctx.state.accounts.get(appAccAddr);
-      if (acc) { acc.amount = 1000000n; }
+      if (acc) {
+        acc.amount = 1000000n;
+      }
 
-      const teal = `
+      const teal =
+        `
```

Comment:
is your editor doing this?

Reply:
> is your editor doing this?

Yes, It's automatic, maybe I should disable it. I remember husky auto format for us but now it not. How about it on your site?